servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl

```diff
@@ -1256,21 +1256,37 @@ void fragment_shader(in SceneData scene_data) {
 #ifndef USE_SHADOW_TO_OPACITY
 
 #ifdef ALPHA_SCISSOR_USED
+#ifdef MODE_RENDER_MATERIAL
+	if (alpha < alpha_scissor_threshold) {
+		alpha = 0.0;
+	} else {
+		alpha = 1.0;
+	}
```

Comment:
I recall there's a condition-less way of achieving this, which should be faster as this condition isn't static (i.e. not all pixels will evaluate to the same value). I remember this using `saturate()` but that's HLSL terminology...

Reply:
`alpha = step(alpha_scissor_threshold, alpha);` should work, right?

Reply:
`step` would indeed implement the same logic, but they will both compile down to the same instruction AFAIK. I don't think there is any benefit in using step vs being more explicit about what you are doing

Reply:
The ISA is identical. https://shader-playground.timjones.io/dc6e4a555ac215a0e0a0d6ccd005888d But weirdly the IL is slightly different. Not sure how that can be as my understanding is that ISA is transformed into IL in a later stage